library/Zend/Validate/File/Extension.php

```diff
@@ -121,9 +121,7 @@ public function setCase($case)
      */
     public function getExtension()
     {
-        $extension = explode(',', $this->_extension);
-
-        return $extension;
+        return ($this->_extension !== null) ? explode(',', $this->_extension) : [''];
```

Comment:
Should we change the value for L137 instead?

Reply:
thanks, fixed as you suggested